OpenRA.Mods.Common/Traits/Attack/AttackFrontal.cs

```diff
@@ -33,26 +39,34 @@ public AttackFrontal(Actor self, AttackFrontalInfo info)
 			this.info = info;
 		}
 
+		public static bool WithinFacingTolerance(int facing, int desiredFacing, int facingTolerance)
```

Comment:
Could you please move this to `Util.cs` with the other facing helpers and call it `FacingsWithinTolerance`?